examples/icd10-react-ts/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "icd10-react-ts",
```

Comment:
there seems to be little difference between react examples in js and ts, is there good reason to be redundant ?

Reply:
Just to check if out package work same for js and ts react app.